frontend/packages/operator-lifecycle-manager/integration-tests-cypress/tests/operator-install-global.spec.ts

```diff
@@ -17,23 +18,42 @@ xdescribe(`Interacting with a global install mode Operator (${operatorName})`, (
     cy.logout();
   });
 
-  const operatorPkgName = 'portworx-essentials';
-  const operatorID = 'portworx-essentials-community-operators-openshift-marketplace';
+  const catalogSourceName = 'console-e2e';
+  const operatorID = 'portworx-essentials-console-e2e-openshift-marketplace';
   const operatorRow = 'Portworx Essentials';
-  const operatorInstallFormURL = `/operatorhub/subscribe?pkg=${operatorPkgName}&catalog=community-operators&catalogNamespace=openshift-marketplace&targetNamespace=`;
+  const operatorPkgName = 'portworx-essentials';
+  const operatorInstallFormURL = `/operatorhub/subscribe?pkg=${operatorPkgName}&catalog=${catalogSourceName}&catalogNamespace=openshift-marketplace&targetNamespace=`;
   const operatorInstance = 'StorageCluster';
   const openshiftOperatorsNS = 'openshift-operators';
   const operandLink = 'portworx';
 
+  it('installs the catalogsource', () => {
+    cy.log('navigate to OperatorHub > Sources');
+    cy.visit('/k8s/cluster/config.openshift.io~v1~OperatorHub/cluster/sources');
+    cy.log('navigate to the CatalogSource creation form');
+    cy.byTestID('item-create').click();
+    cy.byTestID('create-catalogsource-title').should('exist');
+    cy.byTestID('catalog-source-name').type(catalogSourceName);
+    cy.byTestID('catalog-source-image').type(
+      'quay.io/operator-framework/upstream-community-operators@sha256:9243fc9cfdbe3df502772b92abdb0b4794adb4898833fb0a8a5c2f34f6e7ff4e',
+    );
+    cy.byTestID('save-changes').click();
+    cy.log('verify the catalogsource is READY');
+    cy.visit('/k8s/cluster/config.openshift.io~v1~OperatorHub/cluster/sources');
+    cy.byLegacyTestID(catalogSourceName).should('exist');
+    cy.byTestID(`${catalogSourceName}-status`).should('contain', 'READY');
+    // cy.wait(5000);
+    cy.log('verify the packagemanifest has been created');
+    cy.visit('/k8s/ns/openshift-marketplace/packages.operators.coreos.com~v1~PackageManifest');
+    listPage.filter.byName(operatorPkgName);
+    listPage.rows.shouldExist(operatorName);
```

Comment:
So is this where it fails?  Times out after 30 seconds (the default timeout)?  If so, you might want to add a longer timeout to `listPage.rows.shouldExist` (90 seconds) like we did for [listPage.rows.shouldNotExist](https://github.com/openshift/console/blob/6fc9b6a34e3d69de07f8288ca8ea5e6d3cf253e3/frontend/packages/integration-tests-cypress/views/list-page.ts#L65).

Reply:
If I load the list of `PackageManifests` before all the `PackageManifests` are created, they never will all load (I guess we're not watching them here?).  I put the wait of 60 seconds in after creation of the `CatalogSource` and everything works as expected.  I realize this isn't ideal, but I'm trying to just get something that works at the moment.

If we had a `CatalogSource` with just the two `Operators` we needed for the tests, that would certainly help, but I'm not sure we want to do that since that would require the creation of another image that someone would have to maintain.

Reply:
Hmm...ok, it might be flaky under slower CI, something to watch for.
If your interested :-)  https://clearmeasure.com/polling-a-page-in-cypress/

Reply:
> If I load the list of `PackageManifests` before all the `PackageManifests` are created, they never will all load (I guess we're not watching them here?).

@spadgett, any idea why `/k8s/ns/openshift-marketplace/packages.operators.coreos.com~v1~PackageManifest` is not dynamically updating?  Seems like a bug?

Reply:
Implemented the polling technique @dtaylor113 mentioned.